apis/python/notebooks/tutorial_soma_append_mode.ipynb

```diff
@@ -673,6 +686,408 @@
+            "Wrote   /tmp/append-example-20240517-204944/ms/RNA/var\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Writing /tmp/append-example-20240517-204944/ms/RNA/X/data\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944/ms/RNA/X/data\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944/obs\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944/ms/RNA/var\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Writing /tmp/append-example-20240517-204944/ms/RNA/X/data\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944/ms/RNA/X/data\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944/obs\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944/ms/RNA/var\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Writing /tmp/append-example-20240517-204944/ms/RNA/X/data\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944/ms/RNA/X/data\n"
+          ]
+        },
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Wrote   /tmp/append-example-20240517-204944\n"
+          ]
+        }
+      ],
+      "source": [
+        "rd2 = tiledbsoma.io.register_anndatas(\n",
+        "    experiment_uri=None,  # new Experiment, from scratch\n",
+        "    adatas=new_ads,\n",
+        "    measurement_name=measurement_name,\n",
+        "    obs_field_name=\"obs_id\",\n",
+        "    var_field_name=\"var_id\",\n",
+        ")\n",
+        "\n",
+        "for new_ad in new_ads:\n",
```

Comment:
Cell split here.

Then:

Now that we've gotten the registrations for all the input AnnData objects, we can ingest them.

Note:

* Here we ingest them sequentially, in the same order as above.
* But we could also ingest them in any shuffled order.
* Or we could have multiple workers in ingest them in parallel, one worker per AnnData object, as long as the registration data are passed to each worker.

Reply:
done